contracts/mailbox/src/state.rs

```diff
@@ -25,3 +27,77 @@ pub const MESSAGE_TREE: Item<MerkleTree> = Item::new(MESSAGE_TREE_KEY);
 
 pub const MESSAGE_PROCESSED_PREFIX: &str = "message_processed";
 pub const MESSAGE_PROCESSED: Map<Vec<u8>, bool> = Map::new(MESSAGE_PROCESSED_PREFIX);
+
+pub fn assert_owner(owner: Addr, sender: Addr) -> Result<(), ContractError> {
+    if owner != sender {
```

Comment:
i suggest to use pointer instead of moving values

Reply:
```
pub fn assert_owner(owner: &Addr, sender: &Addr) -> Result<(), ContractError> {
    if owner != sender {
```
like this?

Reply:
Yes!

Reply:
Yes!